example/build-with-buf/Cargo.toml

```diff
@@ -14,10 +14,13 @@ proto_full = ["example"]
 
 [dependencies]
 bytes = "1.1.0"
-prost = "0.13.1"
+prost = "0.13.4"
 pbjson = "0.7"
 pbjson-types = "0.7"
 serde = "1.0"
-tonic = { version = "0.12", features = ["gzip"] }
+tonic = { version = "0.13", features = ["gzip"] }
+
+[patch.crates-io]
+tonic = { git = "https://github.com/hyperium/tonic", branch = "master", revision = "5e9a5bc" } # tonic v0.13 (not yet released)
```

Comment:
Since tonic v0.13 has just been released, I suppose this section I originally added would be no longer necessary.

Reply:
Thank you for catching that one. I overlooked it because I was focused on the main crate's toml. Removed in [9f24563](https://github.com/neoeinstein/protoc-gen-prost/pull/119/commits/9f24563aa4eb400975bc29b5cd17a79d5d2eaa26)